### DISCUSSION TITLE
Option to tick all options on MD writer options page?

### DISCUSSION THREAD

**Discussion by mastercodeon314** [dnSpyEx/dnSpy#136] [Ideas]
![unknown-3](https://user-images.githubusercontent.com/78676320/194572731-51427408-9cf0-4376-8560-9c622267d2b7.png)
I would love to see an option added in the settings that auto checks all the options on the MD writer options tab when saving a module. This would be a very time saving option to have for people like me who regularly save obfuscated assemblies. If this isn't something that could be pushed to the next release, where could I find the code to change to allow that to happen?

**Comment by ElektroKill** [comment]
Hi, I will work on making this feature a part of the next release! It does not look hard to implement :p

**Reply by mastercodeon314** [reply]
Sweeeet, that'd be awesome! Could you give me an idea on where I would find the code to make that happen? I've tried for months now to find the code associated with that window. I want to patch my dnspy to make that happen till the next release comes out
